weasyl/test/login/test_authenticate_bcrypt.py

```diff
@@ -0,0 +1,140 @@
+# This Python file uses the following encoding: utf-8
+"""
+Test suite for: login.py::def authenticate_bcrypt(username, password, session=True):
+"""
+import json
+import web
+
+from weasyl.test import db_utils
+from weasyl import login, macro, media
+from weasyl import define as d
+
+
+# Main test password
+raw_password = "0123456789"
+
+
+class Bag(object):
+    def __init__(self, **kw):
+        for kv in kw.items():
+            setattr(self, *kv)
+
+
+def configure_moderators(mod_user_id):
```

Comment:
In this situation, no. I tried both passing it in as a frozenset(), array, and a string, and each time it fails. https://paste.weasyldev.com/show/hM8LMdbqhTQgz3CGx43D/

The ``staff.FOO`` entries are blank in ``conftest.py``, so recalling ``configure_libweasyl()`` to update the frozenset()'s is required in this situation, at least from where I am sitting. If there's some other method, let me know and I'll use it.